templates/test/ci/prow-ci-version-windows/kustomization.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: default
+resources:
+  - ../prow-windows
```

Comment:
I noticed this cluster config contains both Linux and Windows worker nodes. Is this intended?

Reply:
It's build from the base windows template includes Linux nodes. I think the base template should include Linux node. AFAIK there isn't a way to remove resources completely: https://kubectl.docs.kubernetes.io/faq/kustomize/eschewedfeatures/#removal-directives

I guess we could set the replica count to 0?